test_rigs/unit-tests/configs/cisco_misc

```diff
@@ -36,6 +36,8 @@ username abc
 username rancid attributes
  vpn-group-policy GroupPolicy_NOC-VPN
  group-lock value NOC-VPN
+username bob1 password 1 1234
+username bob2 privilege 1 password 1 1234
```

Comment:
@arifogel : separately, we should make sure this information ends up in the data model, too.